src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/SemanticTokensEditCapability.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using OmniSharp.Extensions.LanguageServer.Protocol.Client.Capabilities;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Semantic.Interfaces
```

Comment:
Does this need to go under the models directory to match the [`SemanticTokensCapability`](https://github.com/dotnet/aspnetcore-tooling/blob/36b17bb2761c2920705da7bb766cbf9bdc763d65/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/SemanticTokensCapability.cs)?

```suggestion
namespace Microsoft.AspNetCore.Razor.LanguageServer.Semantic
```